awc/src/ws.rs

```diff
@@ -70,9 +70,14 @@ impl WebsocketsRequest {
         <Uri as TryFrom<U>>::Error: Into<HttpError>,
     {
         let mut err = None;
-        let mut head = RequestHead::default();
-        head.method = Method::GET;
-        head.version = Version::HTTP_11;
+
+        #[allow(clippy::field_reassign_with_default)]
+        let mut head = {
```

Comment:
Necessary because `flags` field is private and doesn't work with clippy's suggestion here.

Arguably a bug in clippy.

Reply:
Yeah, seems a known FP: https://github.com/rust-lang/rust-clippy/issues/6344